source/mus_control_module.f90

```diff
@@ -151,6 +190,32 @@ subroutine mus_init_control( controlRoutine, me, minLevel, maxLevel )
       if ( minLevel == maxLevel ) then
         me%do_computation => do_fast_singleLevel
         write(logUnit(5),"(A)") "Select fast single level control routine."
+
+        ! Choose control routine for coupled LBM-DEM simulations of particulate 
+        ! flows.
+        select case( trim( particle_kind) )
+        case ('MEM')
+          me%do_computation_particles => do_fast_singleLevel_particles_MEM
+          write(logUnit(5),"(A)") "Select fast single level control routine for ", &
+            & " fully coupled MEM particles"
+        case ('DPS')
+          ! TV: use alternate compute routine for particles!
+          me%do_computation_particles => do_fast_singleLevel_particles_DPS
+          write(logUnit(5),"(A)") "Select fast single level control routine for ", &
+            & " fully coupled DPS particles using the Generalized Navier-Stokes equations"
+        case ('DPS_twoway')
+          me%do_computation_particles => do_fast_singleLevel_particles_DPS_twoway
+          write(logUnit(5),"(A)") "Select fast single level control routine ", &
+            & "for two-way coupled DPS particles (neglecting the effect of local ", &
+            & "fluid volume fraction )."
+        case ('DPS_oneway')
+          me%do_computation_particles => do_fast_singleLevel_particles_DPS_oneway
+          write(logUnit(5),"(A)") "Select fast single level control routine ", &
+            & "for one-way coupled DPS particles."
+        case ('default')
+          write(logUnit(5),"(A)") "Particulate flow simulations disabled!"
+          me%do_computation_particles => null()
+        end select
```

Comment:
This is weird, I'd expect the name here to be 'none' or something like that, or should this be:

```suggestion
        case default
          write(logUnit(5),"(A)") "Particulate flow simulations disabled!"
          me%do_computation_particles => null()
        end select
```
to capture all unknown strings to disable the particles?